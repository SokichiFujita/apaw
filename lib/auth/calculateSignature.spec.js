const calculateSignature = require('./calculateSignature');

describe('calculateSignature', () => {
  it('should return a hexidecimal string', () => {
    const calculatedSignature = calculateSignature({
      region: 'region',
      secret: 'secret',
      service: 'service',
      shortDate: '20200308',
      stringToSign: 'string to sign',
      terminationString: 'terminationString',
    });
    expect(calculatedSignature).toMatchSnapshot();
  });
});
