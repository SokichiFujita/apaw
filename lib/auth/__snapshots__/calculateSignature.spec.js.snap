// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calculateSignature should return a hexidecimal string 1`] = `"793438099fec26831f2fb890c2f0da8b51952f324ae5b9c35cf1630a3230c9ce"`;
